lib/chipmunk/incoming_storage.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Chipmunk
+  # IncomingStorage is a factory for proxies to storage of incoming deposits.
+  class IncomingStorage
+
+    # Create an IncomingStorage instance.
+    def initialize(volume:)
+      @volume = volume
+    end
+
+    def for(package)
+      raise Chipmunk::PackageAlreadyStoredError, package if package.stored?
```

Comment:
Parallel issue to below - not sure we need this as `stored?` will eventually go away, and asking the wrong thing for the package will raise an error later on anyway.

Reply:
Yup... It probably falls away after a while. This is basically a programmer error while package is still pulling double duty.